### PR TITLE
Update testing.rst

### PR DESCRIPTION
**Pull request by gabru-md**
Referencing #824 

There were some typing errors in the initial `testing.rst` as provided in PR #824 therefore I've updated `testing.rst` for the same.

Thanks :D

### CODE REVIEW

File: docs/testing.rst [hunk 1]
```diff
@@ -51,5 +51,35 @@ performance by running the following command.
 .. sourcecode:: sh
 
    python tests/benchmarks.py
+   
+Makefile Utility
+----------------
+
+Makefiles are a simple way to perform code compilation on ``Linux platforms``.
+
+We often forget to compile or to run all tests whenever we make any change in existing files,
```
Comment (gunthercox) [inline docs/testing.rst]: > We often forget to compile

Not sure if this is accurate, I don't believe there is anything in ChatterBot that requires compiling.

Comment (vkosuri) [inline docs/testing.rst]: I think it can write like this

``We often forgot to build docs, run nosetes or Django tests``

Comment (gabru-md) [inline docs/testing.rst]: okay! 
Let me do the changes.
:D

**Comment by gabru-md** [comment]
Updates sir ?

**Pull request merged**